React/CoreModules/RCTAlertController.m

```diff
@@ -20,7 +20,23 @@ @implementation RCTAlertController
 - (UIWindow *)alertWindow
 {
   if (_alertWindow == nil) {
-    _alertWindow = [[UIWindow alloc] initWithFrame:RCTSharedApplication().keyWindow.bounds];
+
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_13_0) && \
+    __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
+    if (@available(iOS 13.0, *)) {
+      for (UIScene *scene in RCTSharedApplication().connectedScenes) {
+        if (scene.activationState == UISceneActivationStateForegroundActive && [scene isKindOfClass:[UIWindowScene class]]) {
```

Comment:
Why the check `[scene isKindOfClass:[UIWindowScene class]]` is needed? Can `scene` be of something different from a `UIWindowScene class`?

Reply:
For app supports CarPlay, `RCTSharedApplication().connectedScenes` will include CarPlay scenes such as  `CPTemplateApplicationSceneDelegate` instance